server_apps/jenkins/jobs/UniProt-Secondary-Term-Load/config.xml

```diff
@@ -34,7 +34,7 @@
   <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
   <triggers>
     <hudson.triggers.TimerTrigger>
-      <spec></spec>
+      <spec/>
```

Comment:
This is not going to work if it should get scheduled with a cron expression. I assume this job is always meant to be run manually?

Reply:
Yes, I intend it to only be run manually. Perhaps we could configure it to run automatically at some point if we feel comfortable with that.

Reply:
In order to get this onto trunk, I'm going to merge this into main. I created another pull request against release-1152 so we can continue the discussion there if you have any more concerns before approving.